build/common.sh

```diff
@@ -15,12 +17,12 @@ cloneRepos() {
 	for prereqrepo in ${UTIL_REPOS}; do
 		if [ ! -d ${prereqrepo} ]; then
 			echo "Cloning ${prereqrepo} ..."
-			git clone --quiet https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${COMPONENT_ORG}/${prereqrepo}.git || exit 1
+			git clone --quiet https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${COMPONENT_ORG}/${prereqrepo}.git ${prereqrepo} || exit 1
 		fi
 	done
 	if [ ! -d "${COMPONENT_ORG}" ]; then
 		# Collect repos from main-branch-sync/repo.txt
-		REPOS=$(cat ./main-branch-sync/repo.txt)
+		REPOS=$(cat ${DIR}/main-branch-sync/repo.txt)
```

Comment:
There are some other variable updates here, but this line is the relevant one for this PR.